apps/docs/src/content/docs/reference/sheepdog-utils.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Sheepdog Utils
+description: the utilities object that is available in every task
+---
+
+When creating a task (of any type) there are 2 possible arguments that be accessed, the first is any argument that is passed in when calling `.perform()`, and the second is an object that can be used to extend your task - `SheepdogUtils`.
+
+`SheepdogUtils` offers two properties to extend your task with:
+- `signal` - the `AbortSignal` is available so that you can do any further manipulation you need based on the state of the `AbortSignal`. We won't go into too much detail about this as it is the standard `AbortSignal` interface that is simply passed back from `Sheepdog`.
+- `link` - this is a `Sheepdog` specific function that allows you to bind tasks together. For more information, check out the [Linking Tasks explainer](/explainer/linking-tasks).
+
+```svelte
+<script lang="ts">
+	import { task, type SheepdogUtils } from '@sheepdog/svelte';
+
+	const myTask = task(async (myArgument: any, {signal, link}: SheepdogUtils) => {
```

Comment:
I might be wrong but i don't think you have to type SheepdogUtils...you should be getting autocomplete out of the box.

Reply:
Nope...the more TS is just like JS the Better imho

Reply:
Good to know 👍